website/src/controllers/api-v3/challenges.js

```diff
@@ -86,6 +87,41 @@ api.createChallenge = {
 };
 
 /**
+ * @api {post} /challenges/:challengeId/join Joins a challenge
+ * @apiVersion 3.0.0
+ * @apiName JoinChallenge
+ * @apiGroup Challenge
+ *
```

Comment:
here you should document the challengeId param ` * @apiParam {UUID} challengeId The challenge _id`. I know it's missing in some other routes but I already added it where it's missing in my branch